tracer/src/Datadog.Trace/Iast/Aspects/System/StringAspects.cs

```diff
@@ -16,7 +16,7 @@
 namespace Datadog.Trace.Iast.Aspects.System;
 
 /// <summary> String class aspects </summary>
-[AspectClass("mscorlib,netstandard,System.Private.CoreLib,System.Runtime", AspectFilter.StringOptimization)]
+[AspectClass("mscorlib,netstandard,System.Private.CoreLib,System.Runtime", AspectFilter.StringOptimization, AspectType.Propagation)]
```

Comment:
This should be unnecessary. Default `AspectType` for `AspectClassAttribute` is already `AspectType.Propagation`

Reply:
Yes, I just wanted to add all the aspect types to make them more readable/clear...